src/lib/util/PrettyPrinter.ts

```diff
@@ -0,0 +1,51 @@
+import {RuleFilter} from '../RuleManager';
+
+/**
+ * @param stringSet Set<string> to stringify
+ */
+export function stringifySet(stringSet: Set<string>): string {
+    const returnArr = [...stringSet];
+    return `[${returnArr.join(',')}]`;
+}
+
+/**
+ * Typically used to print Language-RulePath mapping
+ * @param mapOfSet Map<string, Set<string>> to stringify
+ */
+export function stringifyMapSet(mapOfSet: Map<string, Set<string>>): string {
```

Comment:
This name feels kind of weird. It implies you're stringifying a MapSet, but that's not a real thing. Maybe `stringifySetMappings`, or `MapOfSets` or something like that?

Reply:
I like `stringifyMapOfSets(...)`. Let me change it.